StandardAnalysis/python/Cuts.py

```diff
@@ -761,6 +766,13 @@
     cutString = cms.string("caloNewNoPUDRp5CentralCalo > 50"),
     numberRequired = cms.string(">= 1"),
 )
+# If using MiniAOD only processing, the ecalo cut has to be changed to the CMSSW standard
+if dataFormat == 'MINI_AOD_ONLY_2022_CUSTOM':
```

Comment:
looks like there are double entries here, this should only be the 10GeV cut I assume?

Reply:
There are 3 cuts that used the `caloNewNoPUDRp5CentralCalo`: the nominal one with ecalo < 10 GeV, and two inverted ones, one with ecalo > 10 GeV and another one with ecalo > 50 GeV. I just replicated the three of them to not miss any of the cases. The name s of the cuts are `cutTrkEcalo`, `cutTrkEcaloInv` and `cutTrkEcaloInv50`

Reply:
ah I missed that ok looks good approved